validator/tests/test_context_manager/tests.py

```diff
@@ -1033,6 +1033,54 @@ def test_complex_basecontext_squash(self):
                                          "manager calculated merkle hashes "
                                          "are the same")
 
+    def test_wildcarded_inputs_outputs(self):
+        """Tests the context manager with wildcarded inputs and outputs.
+
+        Notes:
+            1. Create a context with a wildcarded input and output and
+               another non-wildarded input and output.
+            2. Get an address under the wildard and set to both the non-wildcarded
+               address and an address under the wildcard..
+            4. Squash the context and compare to a manually generated state
+               hash.
+        """
+
+        namespaces = [
+            self._create_address('a')[:8],
```

Comment:
I guess this is consistent with the rest of the file, but why is `_create_address` called over and over instead of assigning variables? `address_a = self._create_address('a')` or something like that.

Reply:
So I guess in general, I think `self._create_address('a')` is as readable and informative as `address_a` and saves lines where `address_a = self._create_address('a')` would be. Line 1049 where you comment is different in that `namespace_a` would have more information than `self._create_address('a')[:8]`.  I will take this under advisement, but don't want to have some of the namespaces and addresses using `_create_address` and some not.